docs/install/README.md

```diff
@@ -38,21 +38,38 @@ By default kind clusters run on the same Docker network which means we will have
 # Single Cluster Install with Kustomize
 We will first look at a single cluster install to demonstrate that while K8ssandra Operator is designed for multi-clluster use, it can be used in a single cluster without any extran configuration.
 
-## Create kind cluster
+## Automated Setup
```

Comment:
Not sure if it is a spacing issue but `Automated Setup` does not render as a header.

Reply:
I noticed the same thing and strangely couldn't actually fix it, not sure what's up with that structurally.

Reply:
it's usually a bad space character between the `##`  and the title.
I'll fix this.